MathlibTest/eqns.lean

```diff
@@ -19,10 +19,6 @@ theorem t_def : t = 1 := rfl
 -- this rw causes lean to generate equations itself for t before the user can register them
 theorem t_def' : t = 1 := by rw [t]
 
-/--
-error: There already exist stored eqns for 't'; registering new equations
-will not have the desired effect.
--/
```

Comment:
wait what? Presumably someone who understands this file is happy that there is no longer an error here?